docs/operations/rwx.md

```diff
@@ -1,136 +1,431 @@
+ ```
+
+1. You can also review and confirm that Ondat has successfully provisioned a RWX volume, as defined in the PVC manifest earlier, by running the [Ondat CLI utility as a deployment](/docs/reference/cli/) first, so that you can interact and manage Ondat through `kubectl`. Once deployed, obtain the Ondat CLI utility pod name for later reference.
+
+ ```bash
+ # Get the pod name of the Ondat CLI utility.
+
+ kubectl get pods --namespace storageos | grep "storageos-cli"
+ storageos-cli-578c4f4674-wr9z2                       1/1     Running   0              3m43s
+ ```
+
+ ```bash
+ # Get the volumes in the "ondat-files" namespace using the Ondat CLI.
+ kubectl --namespace=storageos exec storageos-cli-578c4f4674-wr9z2 -- storageos get volumes --namespace=ondat-files
+
+ NAMESPACE    NAME                                      SIZE     LOCATION                                  ATTACHED ON                      REPLICAS  AGE
+ ondat-files  pvc-41f429ff-3fa9-4361-b7bc-c55f869499ae  5.0 GiB  aks-default-15645363-vmss000000 (online)  aks-default-15645363-vmss000000  0/0       2 hours ago
+
+ # Describe the "pvc-4457be86-54a3-4f2b-8326-5d4c8799d48a" volume.
+ kubectl --namespace=storageos exec storageos-cli-578c4f4674-wr9z2 -- storageos describe volume pvc-41f429ff-3fa9-4361-b7bc-c55f869499ae --namespace=ondat-files
+
+ ID                      aad1e4ab-23d2-4a45-9a9b-0305885997ff
+ Name                    pvc-41f429ff-3fa9-4361-b7bc-c55f869499ae
+ Description
+ AttachedOn              aks-default-15645363-vmss000000 (1c9fe8f8-7c33-44b0-802d-875ff4c53fdd)
+ Attachment Type         nfs
+ NFS
+   Service Endpoint      10.224.0.4:25918
+   Exports:
+   - ID                  1
+     Path                /
+     Pseudo Path         /
+     ACLs
+     - Identity Type     hostname
+       Identity Matcher  *
+       Squash            all
+       Squash UID        0
+       Squash GUID       0
+ Namespace               ondat-files (97e1ffeb-34e3-4f18-8ef1-5494f34fe9cd)
+ Labels                  csi.storage.k8s.io/pv/name=pvc-41f429ff-3fa9-4361-b7bc-c55f869499ae,
+                         csi.storage.k8s.io/pvc/name=ondat-files,
+                         csi.storage.k8s.io/pvc/namespace=ondat-files,
+                         storageos.com/nfs/mount-endpoint=10.0.133.137:2049,
+                         storageos.com/nocompress=true
+ Filesystem              ext4
+ Size                    5.0 GiB (5368709120 bytes)
+ Version                 NQ
+ Created at              2022-09-08T12:54:49Z (2 hours ago)
+ Updated at              2022-09-08T14:29:18Z (28 minutes ago)
+
+ Master:
+   ID                    67cae629-7797-444a-828d-22d5c22d3535
+   Node                  aks-default-15645363-vmss000000 (1c9fe8f8-7c33-44b0-802d-875ff4c53fdd)
+   Health                online
+ ```
+
+1. Notice under `Attachment Type         nfs`, there detailed information about the RWX volume is returned.
```

Comment:
is it normal that there are so many spaces between Type and nfs?

Reply:
**the** detailed

Reply:
> is it normal that there are so many spaces between Type and nfs?

I copied it directly from the cli output. After rendering, the spaces are removed.